src/twisted/test/test_task.py

```diff
@@ -534,20 +534,20 @@ def foo(cnt):
         loop.clock = clock
         loop.start(0, now=False)
 
-        clock.advance(0)
+        clock.pump([0] * 2)
         # Loop will block at the third call.
-        self.assertEqual([1, 1], accumulator)
+        self.assertEqual([1] * 2, accumulator)
 
         # Even if more time pass, the loops is not
         # advanced.
-        clock.advance(2)
-        self.assertEqual([1, 1], accumulator)
+        clock.pump([1] * 2)
+        self.assertEqual([1] * 2, accumulator)
 
         # Once the waiting call got a result the loop continues without
         # observing any delay in countCallable.
         deferred.callback(None)
-        clock.advance(0)
-        self.assertEqual([1, 1, 1, 1, 1], accumulator)
+        clock.pump([0] * 4)
+        self.assertEqual([1] * 5, accumulator)
```

Comment:
Just asking.

Is there any advantage in this way of writing the test?

I prefer to have the assertions as "dumb" as possible, to reduce the risk of errors...to the extend of having more repetition.

Reply:
I think you're asking about  `[1, 1, 1, 1, 1]` vs `[1] * 5`?  I think I find `[1] * 5` to be the less risky version.  My eyes slide over `[1, 1, 1, 1, 1]` very easily but the exact number of 1s there is absolutely critical to the correctness of the test.

Probably there is some other pattern altogether that is better than either of these, keeping things simpler than list multiplication but also making it easy to read without making a mistake.  I didn't think very hard about finding such a pattern while working on this.